### ISSUE TITLE
Support SQLite Date stored as integer

### ISSUE THREAD

**Issue by jimmykeen** [fsprojects/SQLProvider#308]
I have SQLite database with DATE column where I store dates as integers:

> INTEGER as Unix Time, the number of seconds since 1970-01-01 00:00:00 UTC.

https://www.sqlite.org/datatype3.html#section_2_2

This causes _"String was not recognized as a valid DateTime"_ error while querying data. Is there any workaround for this issue?

**Comment by jimmykeen** [comment]
I checked some other data access librariers and this (relying on dates being well-formatted date string) is the usual approach with SQLite. So, no-op here.

**Comment by s952163** [comment]
@jimmykeen storing INT and extracting it via a query expression certainly works. You can convert with ToUnixTimeSeconds and FromUnixTimeSeconds in .NET 4.6. if you specifiy INTEGER as column type in SQLite. For example:

```
query {                                         
   for r in table7 do
   where (r.Date1 > (DateTimeOffset(DateTime(2016,12,8,0,0,0)).ToUnixTimeSeconds()) )
   select (DateTimeOffset.FromUnixTimeSeconds r.Date1)                                
   } |> Seq.toList  |> Seq.length
```

Storing DateTime also works (column type DateTime). If you want to use an ISO date string to represent the date look at issue #328. It got fixed in 1.0.32 with a nice workaround.
